docs/src/InvariantTheory/finite_groups.md

```diff
@@ -91,6 +91,14 @@ julia> M2 = matrix(K, [1 0 0; 0 a 0; 0 0 -a-1])
 julia> G = matrix_group(M1, M2)
 Matrix group of degree 3
   over cyclotomic field of order 3
+with 2 generators
+  [0   0   1]
+  [1   0   0]
+  [0   1   0]
+
+  [1   0        0]
+  [0   a        0]
+  [0   0   -a - 1]
 
 julia> IR = invariant_ring(G)
```

Comment:
The printing of the invariant ring should also be adjusted, I just did not yet get to that (CC @joschmitt)